packages/logs/src/boot/startLogs.ts

```diff
@@ -62,31 +66,48 @@ export function doStartLogs(
     onLogEventCollected = (message) => batch.add(message)
   }
 
-  function reportError(error: RawError) {
-    errorLogger.error(
-      error.message,
-      combine(
-        {
-          date: error.startClocks.timeStamp,
-          error: {
-            kind: error.type,
-            origin: error.source,
-            stack: error.stack,
-          },
+  function reportRawError(error: RawError) {
+    const messageContext: Partial<LogsEvent> = {
+      date: error.startClocks.timeStamp,
+      error: {
+        kind: error.type,
+        origin: error.source,
+        stack: error.stack,
+      },
+    }
+    if (error.resource) {
+      messageContext.http = {
+        method: error.resource.method as any, // Cast resource method because of case mismatch cf issue RUMF-1152
+        status_code: error.resource.statusCode,
+        url: error.resource.url,
+      }
+    }
+    logger.error(error.message, messageContext)
+  }
+
+  const LogStatusForApi = {
+    [ConsoleApiName.log]: StatusType.info,
+    [ConsoleApiName.debug]: StatusType.debug,
+    [ConsoleApiName.info]: StatusType.info,
+    [ConsoleApiName.warn]: StatusType.warn,
+    [ConsoleApiName.error]: StatusType.error,
+  }
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏we could move this configuration constant at the top of the file, it would reduce a bit the length of this method